src/Service.GraphQLBuilder/Mutations/UpdateMutationBuilder.cs

```diff
@@ -236,7 +239,7 @@ private static NameNode GenerateInputTypeName(string typeName)
                     new List<DirectiveNode>()));
 
             // Create authorize directive denoting allowed roles
-            List<DirectiveNode> fieldDefinitionNodeDirectives = new();
+            List<DirectiveNode> fieldDefinitionNodeDirectives = new() { new(ModelDirectiveType.DirectiveName, new ArgumentNode("name", dbEntityName)) };
```

Comment:
"name" can be replaced with `ModelDirectiveType.ModelNameArgument` - in all 3 `MutationBuilders`

Reply:
fixed.